src/nft/components/collection/ActivityCells.tsx

```diff
@@ -57,6 +58,15 @@ const AddressLink = styled(ExternalLink)`
   }
 `
 
+const isPurchasableOrder = (orderStatus?: OrderStatus, marketplace?: string): boolean => {
```

Comment:
this is separate from updating the marketplace icons, can we split this out into it's own PR or update the title and description?